download_ffhq.py

```diff
@@ -384,6 +385,7 @@ def run_cmdline(argv):
     parser.add_argument('--timing_window',      help='samples for estimating download eta (default: 50)', type=int, default=50, metavar='LEN')
     parser.add_argument('--chunk_size',         help='chunk size for each download thread (default: 128)', type=int, default=128, metavar='KB')
     parser.add_argument('--num_attempts',       help='number of download attempts per file (default: 10)', type=int, default=10, metavar='NUM')
+    parser.add_argument('--base_directory',     help="directory to save the downloaded files (default: current folder)", default="")
```

Comment:
I'd suggest calling the argument "download_directory" instead, because that makes more sense to the user.

Reply:
Agreed. Download directory is more intuitive. Updated the PR now.